frontend/talentsearch/src/js/main.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import ContextContainer from "@common/components/context/ContextContainer";
+import Toast from "@common/components/Toast/Toast";
+
+import Router from "./components/Router";
+
+import * as messages from "./lang/frCompiled.json";
+
+ReactDOM.render(
+  <>
+    <ContextContainer messages={messages}>
+      <Router />
+    </ContextContainer>
+    <Toast />
+  </>,
+  document.getElementById("app"),
+);
```

Comment:
Hmm, can't seem to reproduce this on in Chrome or Firefox on macos 🤔  (Safari isn't showing favicons in the network tab 😢 )

<img width="646" alt="Screenshot 2022-11-16 at 4 59 30 PM" src="https://user-images.githubusercontent.com/4127998/202304083-17b27031-cecb-4fc8-898b-c28161613102.png">

<img width="1512" alt="Screenshot 2022-11-16 at 5 01 21 PM" src="https://user-images.githubusercontent.com/4127998/202304101-6863fec6-0199-4f96-80e1-15b77ab00fda.png">

Reply:
To me, it looks like when urql aborts an inflight request when it gets a second identical request.  I might poke at it a bit to see if I can figure it out but it's certainly not a blocker.
![image](https://user-images.githubusercontent.com/8978655/202463561-6a76e91f-6191-4c8a-bdb9-de4a252f1374.png)

Reply:
Thanks, that makes sense. Some of the `urql` requests we run in the context providers cause the app to re-render (something I'd like to avoid if we can).

Reply:
Oh, it's probably the 301 that starts it over.  Not sure why I didn't notice that behaviour before.  Probably has nothing to do with this PR.  Nevermind.  :smile:

Reply:
haha, wow. Yeaahh, that is likely it 😛